lib/jsonld_serializers.js

```diff
@@ -257,14 +257,13 @@ class ResourceSerializer extends JsonLdItemSerializer {
     // Add hasVol and hasDate properties
     stmts.hasItemVolumes = false
     stmts.hasItemDates = false
-    // once we switch to RCI, we can remove numItems check.
-    const numItems = this.body.numItems && this.body.numItems[0] || this.body.numItemsTotal[0]
+    const numItems = this.body.numItemsTotal && this.body.numItemsTotal[0] || this.body.numItems && this.body.numItems[0]
```

Comment:
can we jettison the numItems check at this point? or are we holding onto it just in case we need to fall back to DHI at some point?

Reply:
ope i just read your comments. disregard!

Reply:
Need to hold onto it because some older bibs may not have numItemsTotal. (Were they to be reindexed, they would, but that may not happen for a while.)